portfolio/src/main/webapp/script.js

```diff
@@ -20,7 +20,20 @@ function off() {
 }
 // asynchronously fetch content from server
 async function getContent() {
-  const response = await fetch('/data');
-  const quote = await response.text();
-  document.getElementById('content-container').innerText = quote;
+    const response = await fetch('/data');
+    const content = await response.json();
+  
+    const contentListElement = document.getElementById("content-container");
+    contentListElement.innerHTML = '';
+    for(let i=0; i < content.length; i++) {
```

Comment:
nit: I think the style guide wants spaces around the equals sign.  so: let i = 0;

Reply:
oh i see, thanks!